spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -916,6 +930,53 @@ public void setReadCompacted(boolean readCompacted) {
 			this.readCompacted = readCompacted;
 		}
 
+		public Startup getStartup() {
+			return this.startup;
+		}
+
+	}
+
+	public static class Startup {
+
+		/**
+		 * The max time to wait for the container to start.
+		 */
+		private Duration timeout;
+
+		/**
+		 * The action to take when the container fails to start.
+		 */
+		private FailurePolicy onFailure;
```

Comment:
I understand the previous behavior was to continue and log an exception. There's no default value here so it's hard to see if that's still the case. If that's the case, this field should have a default value that matches the default behavior.